xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -40,9 +40,11 @@
         <Parameter Name="stream" Type="System.IO.Stream" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="stream">The output stream.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Resources.Extensions.PreserializedResourceWriter" /> class that writes the resources to the provided stream.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">The <paramref name="stream" /> is not writable.</exception>
```

Comment:
Either "stream is not writable" or "The stream parameter is not writable".

Reply:
```suggestion
        <exception cref="T:System.ArgumentException"><paramref name="stream" /> is not writable.</exception>
```